cloud_auto.yaml

```diff
@@ -79,12 +90,36 @@ Resources:
         UnhealthyThreshold: '2'
         Interval: '5'
         Timeout: '2'
+      AccessLoggingPolicy:
+        S3BucketName: !Ref S3LoggingBucket
+        Enabled: !Ref EnableELBLogs
+        EmitInterval: '5'
       Tags:
       - Key: Name
         Value: 'MyLoadBalancer'
+    DependsOn: LogsBucketPolicy
+
+# Config S3 backet for ELB access logs 
+  S3LoggingBucket:
+    Type: AWS::S3::Bucket
+  LogsBucketPolicy:
+    Type: AWS::S3::BucketPolicy
+    Properties:
+      Bucket: !Ref S3LoggingBucket
+      PolicyDocument:
+        Version: '2008-10-17'
+        Statement:
+        - Sid: ELBAccessLogs20130930
+          Effect: Allow
+          Resource: !Join ['', [!FindInMap [Region2ARNPrefix, !Ref 'AWS::Region',
```

Comment:
Lets simplify this block. Try to change !Join to !Sub

Reply:
fixed